package.json

```diff
@@ -11,6 +16,7 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-modal": "^3.11.2",
```

Comment:
optional: You are using the reactstrap modal, right? This could probably be removed.

Reply:
Okay removed!